lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -96,6 +103,10 @@ defmodule Bamboo.SMTPAdapter do
     raise SMTPError, {reason, detail}
   end
 
+  defp handle_response({:error, detail}) do
+    raise SMTPError, {:not_specified, detail}
+  end
+
```

Comment:
```suggestion
```

Well now you can remove this 😅

Reply:
I think you didn't read my response on the comment above, I'm keeping this because `gen_smtp` **can** send us an error tuple with only detail, that's why I'm keeping it.

Reply:
You are right, sorry 😅

Reply:
There is no reason to be sorry :)   I answered on a closed outdated comment, is pretty normal to skip it.